harmonic/troubleshooting.md

```diff
@@ -0,0 +1,178 @@
+# Troubleshooting
+
+## Gazebo libraries are not found
+If you see this error message:
+
+```bash
+I cannot find any available 'gz' command:
+	* Did you install any Gazebo library?
+	* Did you set the GZ_CONFIG_PATH environment variable?
+	    E.g.: export GZ_CONFIG_PATH=$HOME/local/share/gz
+```
+
+You should set up the environment variable:
+
+```
+# replace <path_to_install_dir> to your Gazebo installation directory
+GZ_CONFIG_PATH=<path_to_install_dir>/share/gz/
```

Comment:
The garden version has `GZ_CONFIG_PATH=/usr/local/share/gazebo/`. Should that be updated as well?

Reply:
yep updated and ported more fixes to Garden in [937916a](https://github.com/gazebosim/docs/pull/324/commits/937916a493ee0ed5f721e5ab648f62c5d4717a3d)